data/wanderer dire storm.txt

```diff
@@ -0,0 +1,136 @@
+
+ship "Dire Storm"
+	sprite "ship/dire storm"
+	attributes
+		category "Heavy Warship"
+		licenses
+			"Wanderer Military"
+		"cost" 40500000
+		"shields" 54128
+		"hull" 48962
+		"required crew" 72
+		"bunks" 237
```

Comment:
That's a lot of bunks. Is this a passenger ship?

Reply:
It's a deep space exploration cruiser. To boldly go where no bird has gone before.

Reply:
They literally have the need to explore dangerous, unexplored parts of the galaxy every time they go through the Eye. Their need to explore the new frontier literally forms the basis of the wanderer campaign, first defending their old worlds long enough for the eye to open, and then taming and settling the wild reaches. And their science doesn't take place exclusively in safe, sanitary laboratories; it literally takes place around and with life forms that can and will eat them if given the chance.

Reply:
Yeah, and that's why they have a strong military fleet. They don't need to press their scientific staff into military service. They can just protect them with real warships.

Reply:
The Wanderers only seem to begin building new ships as they need them. If your argument for the Dire Storm is that the Wanderers need to explore every time the Eye is opened, then they don't need the DIre Storm here because of how the story already has the player doing the exploration for the Wanderers.

Reply:
Player has successfully not died after jumping into a number of systems. However, if the wanderers intend to settle there, their civilians need better protection than is the norm for their species because a random fleet of rabid drones -- possibly led by an alpha or a korath raiding fleet -- could literally show up at any minute. Even after the wanderer mid campaign is resolved, they have good reason to protect their civilian science efforts and are culturally not well equipped to have war fleets on standby.